content/python/concepts/functions/terms/isinstance/isinstance.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'isinstance()'
+Description: 'Returns `True` if object is the specified type. Otherwise function will return `False`.'
+Subjects:
+  - 'Computer Science'
+  - 'Python'
+  - 'Engineering'
+  - 'Programming'
+  - 'Web Dev'
+  - 'Data Science'
+  - 'Data Analytics'
+Tags:
+  - 'Python'
+  - 'Functions'
+  - 'Built-in'
+  - 'Boolean'
+  - 'Objects'
+  - 'Types'
+  - 'TypeError'
+  - 'Parameters'
+  - 'Class'
+CatalogContent:
```

Comment:
This needs to be edited. Look at [this list](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md) for the proper slugs.